kits/know-thy-person/prompts/know-thy-person_instructor-llmnode-946_user_1.md

```
@@ -0,0 +1,8 @@
+Person anchor:
+- Name: {{Resolve.output.name}}
+- Email domain: {{Resolve.output.domain}}
+- Company (from email): {{Resolve.output.company}}
+- User-provided link (authoritative if present): {{triggerNode_1.output.person_context}}
+Company website content (crawled — PRIMARY authoritative source):
+{{Firecrawl.output}}
+Search results (JSON): {{webSearchNode_194.output.output}}
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)